src/Nethermind/Directory.Packages.props

```diff
@@ -68,6 +69,7 @@
     <PackageVersion Include="Seq.Api" Version="2023.3.0" />
     <PackageVersion Include="SharpCompress" Version="0.34.1" />
     <PackageVersion Include="Shouldly" Version="4.2.1" />
+    <PackageVersion Include="Snappier" Version="1.1.1" />
```

Comment:
Is it faster or better than then already used `Snappy.Standard`?

Reply:
The problem is that era1 uses [framing format ](type: [0x01, 0x00]
data: snappyFramed(ssz(SignedBeaconBlock))) which the existing lib does not support